### DISCUSSION TITLE
Great looking tool! Use with udm-utilities?

### DISCUSSION THREAD

**Discussion by marceldarvas** [ep1cman/unifi-protect-backup#9] [Ideas]
Since [udm-utilities](https://github.com/boostchicken-dev/udm-utilities) allows you to install python3 and rclone, could this be run directly via podman?

**Comment by ep1cman** [comment]
udm-utilities is the next thing I would like to look into. This is a fairly low overhead application so should run there no problem.

I see there being two approaches to it:

1) Install the container via podman, this would not require any external dependencies since the container includes `python3` and `rclone` inside

2) Install `python3` and `rclone` on the UDM and run `unifi-protect-backup` directly.

I personally prefer option 1 as its much easier to clean up/update etc. This weekend I will look into how udm-utilities works and get this added.

**Reply by marceldarvas** [reply]
Watch for the podman update script. With the 1.11 UniFi update it has broken that feature, so don't try that.
However it's been great with Cloudflare DDNS upgrades, SSL and I'm considering HAproxy.
